frontend/pages/index.js

```diff
@@ -23,52 +113,120 @@ const HeroSection = () => {
           </div>
         </div>
 
-        {/* Right Content - Illustration Placeholder */}
         <div className="hidden md:flex justify-center">
-          <img
-            src="/images/Teardrop-bg.png"
-            alt="Water Network"
-            className="w-full max-w-md"
-          />
+          <div className="animate-float relative w-[360px] h-[360px]">
```

Comment:
Very nice! It makes the file very long though. Please separate this out into a separate component.